client/asset/btc/btc.go

```diff
@@ -565,6 +565,15 @@ func (btc *ExchangeWallet) SyncStatus() (bool, float32, error) {
 	return true, 1, nil
 }
 
+// ValidateAddress returns wether given address belongs to wallet or not
```

Comment:
wether -> whether.  But maybe rephrase like `ValidateAddress indicates if an address belongs to the wallet.`